### PR TITLE
Align the description of TiFlash Grafana dashboard with other components

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### What is changed, added or deleted? (Required)

The description of TiFlash's Grafana dashboard is not aligned with other components. And we refactor the TiFlash's dashboard in v4.0.5.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): https://github.com/pingcap/tidb-ansible/pull/1343

### CODE REVIEW

File: tiflash/monitor-tiflash.md [hunk 1]
```diff
@@ -5,32 +5,66 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
-| tiflash_raft_read_index_count  | coprocessor 触发 read_index 请求的次数，等于一个 coprocessor 触发的 Region 总数   |
-| tiflash_raft_read_index_duration_seconds | read_index 消耗的时间，主要消耗时间在于和 leader 的交互和重试时间  |
-| tiflash_raft_wait_index_duration_seconds | wait_index 消耗的时间，即拿到 read_index 请求后，等待 local index >= read_index 所花费的时间 |
+使用 TiDB Ansible 或 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)，监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+
+目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node\_exporter、Overview 等。
+
+TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**TiFlash-Proxy-Details**。通过面板上的指标，可以了解 TiFlash 当前的状态。其中 **TiFlash-Proxy-Summary**、**TiFlash-Proxy-Details** 主要为 TiFlash 的 Raft 层信息，其监控指标信息可参考 [TiKV 监控指标详解](/grafana-tikv-dashboard.md)
+
+> 注：低版本的 TiFlash 监控信息较不完善，如有需要推荐使用 v4.0.5 或更高版本的 TiDB 集群。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:14]:
```suggestion
> **注意：**
>
> 低版本的 TiFlash 监控信息较不完善，如有需要推荐使用 v4.0.5 或更高版本的 TiDB 集群。
```

File: tiflash/monitor-tiflash.md [hunk 2]
```diff
@@ -5,32 +5,66 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
+- Available size：每个 TiFlash 实例的可用的存储空间的大小。
+- Capacity size：每个 TiFlash 实例的存储容量的大小。
+- Uptime：自上次重启以来 TiFlash 正常运行的时间。
+- Memory：每个 TiFlash 实例内存的使用情况。
+- CPU Usage：每个 TiFlash 实例 CPU 的使用率。
+- FSync OPS：每个 TiFlash 实例每秒进行 fsync 操作的次数。
+- File Open OPS：每个 TiFlash 实例每秒进行 open 操作的次数。
+- Opened File Count：当前每个 TiFlash 实例打开的文件句柄数。
+
+> 注：
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:30]:
```suggestion
```

File: tiflash/monitor-tiflash.md [hunk 3]
```diff
@@ -5,32 +5,66 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
+> Store size, FSync OPS, File Open OPS, Opened File Count 目前仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:31]:
```suggestion
> **注意：**
>
> Store size、FSync OPS、File Open OPS、Opened File Count 目前仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
```

File: tiflash/monitor-tiflash.md [hunk 4]
```diff
@@ -5,32 +5,66 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
+- Response Bytes/Seconds：所有 TiFlash 实例应答总字节数。
+
+## DDL
+
+- Schema Version：每个 TiFlash 实例目前缓存的 schema 版本。
+- Schema Apply OPM：所有 TiFlash 实例每分钟 apply 同步 TiDB schema diff 的次数。diff apply 是正常的单次 apply 过程，如果 diff apply 失败，则 failed apply +1，并回退到 full apply，拉取最新的 schema 信息以更新 TiFlash 的 schema 版本。
+- Schema Internal DDL OPM：所有 TiFlash 实例每分钟执行的内部 DDL 次数。
+- Schema Apply Duration：所有 TiFlash 实例 apply schema 消耗的时间。
+
+## Storage
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:49]:
```suggestion
## Storage

```

File: tiflash/monitor-tiflash.md [hunk 5]
```diff
@@ -5,32 +5,66 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
+- Internal Tasks OPS：所有 TiFlash 实例每秒进行内部数据整理任务的次数。
+- Internal Tasks Duration：所有 TiFlash 实例进行内部数据整理任务消耗的时间。
+- Page GC Tasks OPM：所有 TiFlash 实例每分钟进行 Delta 部分数据整理任务的次数。
+- Page GC Tasks Duration：所有 TiFlash 实例进行 Delta 部分数据整理任务消耗的时间分布。
+- Disk Write OPS：所有 TiFlash 实例每秒进行磁盘写入的次数。
+- Disk Read OPS：所有 TiFlash 实例每秒进行磁盘读取的次数。
+- Write flow：所有 TiFlash 实例磁盘写操作的流量。
+- Read flow：所有 TiFlash 实例磁盘读操作的流量。
+
+> 注：
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:63]:
```suggestion
```

File: tiflash/monitor-tiflash.md [hunk 6]
```diff
@@ -5,32 +5,66 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
+> 目前这部分监控指标仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:64]:
```suggestion
> **注意：**
>
> 目前这部分监控指标仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
```

File: tiflash/monitor-tiflash.md [hunk 7]
```diff
@@ -5,32 +5,71 @@ aliases: ['/docs-cn/dev/reference/tiflash/monitor/']
...
-| tiflash_raft_wait_index_duration_seconds | wait_index 消耗的时间，即拿到 read_index 请求后，等待 local index >= read_index 所花费的时间 |
+使用 TiDB Ansible 或 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)，监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+
+目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node\_exporter、Overview 等。
+
+TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**TiFlash-Proxy-Details**。通过面板上的指标，可以了解 TiFlash 当前的状态。其中 **TiFlash-Proxy-Summary**、**TiFlash-Proxy-Details** 主要为 TiFlash 的 Raft 层信息，其监控指标信息可参考 [TiKV 监控指标详解](/grafana-tikv-dashboard.md)
+
+> **注意：**
+>
+> 低版本的 TiFlash 监控信息较不完善，如有需要推荐使用 v4.0.5 或更高版本的 TiDB 集群。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:16]: 4.0.5 还未发布

Comment (JaySon-Huang) [reply tiflash/monitor-tiflash.md:16]: 对，相关 PR 已经合了，在发布时会带上。

**Pull request merged**